bagua-opentelemetry/src/exporter/mod.rs

```diff
@@ -0,0 +1,62 @@
+pub mod agent;
+
+use crate::exporter::agent::{AgentAsyncClientHTTP, BaguaBatch, BaguaSpan};
+use async_trait::async_trait;
+use opentelemetry::{sdk::export::trace, Key};
+use reqwest::StatusCode;
+use std::time::UNIX_EPOCH;
+
+#[derive(Debug)]
+pub struct Exporter {
+    pub uploader: AgentAsyncClientHTTP,
+}
+
+#[async_trait]
+impl trace::SpanExporter for Exporter {
+    async fn export(&mut self, batch: Vec<trace::SpanData>) -> trace::ExportResult {
```

Comment:
We need to verify the export function is indeed run in a async runtime that does not block the main thread (training). Please confirm this is the case (for example by sleeping multiple seconds in this function to see if the training is slowed down)

Reply:
Let me test

Reply:
Checked, do not block the main thread